admin/gulpfile.js

```diff
@@ -11,78 +9,78 @@ const standard = require('gulp-standard')
 
 // These files will get uglified and packaged into `app.js`
 const jsBundleFiles = [
-  './assets/javascripts/jquery-1.12.4.js',
```

Comment:
you have added extra spaces which are not acceptable by the standardJS linter.
Can you fix them please.